homepairs/homepairsUI/src/Elements/Forms/GoogleInputForm.tsx

```diff
@@ -0,0 +1,179 @@
+import { GoogleAutoComplete } from 'react-native-google-autocomplete';
+import React from 'react';
+import { Text, View, TextInput, ViewStyle, StyleSheet, ScrollView } from 'react-native';
+import {LocationItem} from 'homepairs-components';
+import {HelperText} from 'react-native-paper';
+import {FontTheme} from 'homepairs-base-styles';
+import * as BaseStyles from 'homepairs-base-styles';
+
+export type InputFormProps = {
+    key?: any;
```

Comment:
I understand that this has default types, but still, shouldn't you never be calling this without a key? or is this just to make testing easier?